commitlint.config.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   extends: ["@commitlint/config-conventional"],
+  ignores: [(message) => /^Bumps \[.+]\(.+\) from .+ to .+\.$/m.test(message)],
```

Comment:
In this way we won't lint dependabot's commits at all, right?
What if the dependabot conf is changed to not use conventional commits anymore?
In case of a breaking dependabot PR, we wouldn't consider it in the changelog and for versioning 😞

Can we exclude dependabot commits for just the length rule?